### DISCUSSION TITLE
Digiham Integration

### DISCUSSION THREAD

**Discussion by robertlynch3** [jketterl/codecserver#7] [Q&A] [answered]
Can [Digiham](https://github.com/jketterl/digiham) be used instead of a traditional USB device?

**Comment by jketterl** [comment, marked as answer]
What is a "traditional USB device"? A mouse, a keyboard, a hard drive? I suppose you're rather referring to AMBE USB devices. If that's what you mean, the answer is no. Digiham only extracts the compressed AMBE audio data. It has a separate component called "mbe_synthesizer" that implements a codecserver client, which can be used to decode the audio data. You can find examples in the digiham repository.
